api_v1/openapi.yml

```diff
@@ -858,3 +910,84 @@ components:
             - "2023-03-01T14:05:52+01:00"
         result:
           $ref: '#/components/schemas/ProductDetails'
+
+    IdentifierReference:
+      type: object
+      description: A reference for a known Identifier, and how to look up any known Products for a given Identifier
+      properties:
+        name:
+          type: string
+          description: A known identifier to Endoflife.date.
+          examples:
+            - purl
+            - cpe
+            - repology
+        uri:
+          type: string
+          format: uri
+          description: The URI to be queried to retrieve the list of Products for a given Identifier
+          examples:
+            - purl
+            - cpe
+            - repology
+      required:
+        - name
+        - uri
+
+    RetrieveIdentifiersResponse:
+      type: object
+      description: A response containing all known Identifiers.
+      properties:
+        schema_version:
+          type: string
+          description: Version of this schema.
+          example: 1.0.0
+        last_modified:
+          type: string
+          format: date-time
+          description: The time this product was last modified.
+          example: 2023-03-01T14:05:52+01:00
+        result:
+          type: array
+          description: Known identifiers (purl, repology, cpe...) to Endoflife.date.
+          items:
+            $ref: '#/components/schemas/IdentifierReference'
+      required:
+        - schema_version
+        - last_modified
+        - result
+
+    RetrieveIdentifierResponse:
```

Comment:
To keep the naming close to what already exists I suggest to name this `IdentifierListResponse`.